hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/ECBlockOutputStreamEntry.java

```diff
@@ -290,6 +290,14 @@ private BlockID underlyingBlockID() {
     return blockOutputStreams[0].getBlockID();
   }
 
+  public List<ECBlockOutputStream> getDataWriteFailedStreams() {
```

Comment:
Wouldn't streamsWithWriteFailure() and streamsWithPutBlockFailure() be better names for these? For me it seems to be easier to get what the actual result is, but maybe it is just my taste of naming, so I can live with these names as well.

Reply:
Sure, I updated the method names.